### PR TITLE
fix: hotfix for db connection issue

### PR DESCRIPTION
**Pull request by codebender37**
- removed disconnect_db() in migrate_from_db
- increase dendrite forward timeout

### CODE REVIEW

**Review by karootplx — APPROVED** [review]
lgtm

File: neurons/validator.py [hunk 1]
```diff
@@ -110,6 +110,8 @@ def __init__(self):
         )
         self.check_registered()
 
+        init_wandb(config=self.config, my_uid=self.uid, wallet=self.wallet)
+
         # Run score migration before loading state
         migration_success = self.loop.run_until_complete(ScoreStorage.migrate_from_db())
         if not migration_success:
```
Comment (jarvis8x7b) [inline neurons/validator.py:117]: add a log for the case migration_success = True

**Review by jarvis8x7b — CHANGES_REQUESTED** [review]

**Review by jarvis8x7b — APPROVED** [review]
lgtm

**Pull request merged**